lib/ansible/module_utils/network/vyos/config/static_routes/static_routes.py

```diff
@@ -0,0 +1,410 @@
+        return None
+
+    def _state_replaced(self, want, have):
+        """ The command generator when state is replaced
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        if have:
+            commands.extend(self._state_deleted(want, have))
+        commands.extend(self._state_merged(want, have))
+        return commands
+
+    def _state_overridden(self, want, have):
+        """ The command generator when state is overridden
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+
+        commands = []
+        for have_item in have:
+            af_have = have_item['address_families']
+            for element in af_have:
+                have_routes = element['routes']
+                for route in have_routes:
+                    route_in_want = self.search_route_in_have(want, route['dest'])
+                    if not route_in_want:
+                        commands.append(
+                            self._compute_command(route['dest'], remove=True)
+                        )
+
+        for want_item in want:
+            af_want = want_item['address_families']
+            for element in af_want:
+                want_routes = element['routes']
+                for route in want_routes:
+                    route_in_have = self.search_route_in_have(have, route['dest'])
+                    commands.extend(self._state_replaced(route, route_in_have))
+
+        return commands
+
+    def _state_merged(self, want, have):
+        """ The command generator when state is merged
+
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        commands = []
+        if have:
+            commands.extend(self._render_updates(want, have))
+        else:
+            commands.extend(self._render_set_commands(want))
+        return commands
+
+    def _state_deleted(self, want, have):
+        """ The command generator when state is deleted
+
+        :rtype: A list
+        :returns: the commands necessary to remove the current configuration
+                  of the provided objects
+        """
+
+        commands = []
+        if want:
+            for key, value in iteritems(want):
+                if value:
+                    if key == 'next_hops':
+                        commands.extend(self._update_next_hop(want, have))
+                    elif key == 'blackhole_config':
+                        commands.extend(self._update_blackhole(key, want, have))
+
+        elif have:
+            commands.append(
+                self._compute_command(dest=have['dest'], remove=True)
+            )
+        return commands
+
+    def _render_set_commands(self, want):
```

Comment:
can u give comment for each of these fn. for maintenance.
```suggestion
        def set_config(self, existing_static_routes_facts):
        """ Collect the configuration from the args passed to the module,
            collect the current configuration (as a dict from facts)

        :rtype: A list
        :returns: the commands necessary to migrate the current configuration
                  to the desired configuration
```